samples/cellular/nrf_cloud_rest_device_message/boards/nrf9160dk_nrf9160_ns.overlay

```diff
@@ -0,0 +1,60 @@
+/* Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/ {
+	/* Define 4 monochromatic LEDs connected directly
+	 * to the GPIO to be used with CAF LEDs module
+	 */
+	led0 {
+		compatible = "gpio-leds";
+		label = "LED 1";
```

Comment:
Zephyr has dropped support for the label property, so let's kill that from these device tree bindings